recipe/tiledb-patches/system-ports/abseil/vcpkg.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
can you add an explanation in the PR commit message for all these `vcpkg` related patches?

Reply:
Honestly, I think Ian was trying something there, but those seem unused at this point.

Reply:
you are adding a series of files in the `tiledb-patches/system-ports/<pkg-name>/vcpkg.json`. It would be nice to know what those files are about. If you think these files are unused, then you should remove them. Otherwise some documentation in the PR commit message is necessary.

Reply:
So I spoke with Ian, and here it is:
- Explained in detail here: https://github.com/conda-forge/tiledb-feedstock/tree/main/recipe/tiledb-patches/system-ports
- TLDR: This tells vcpkg to not vendor (download and build) certain dependencies that we have already via conda.
I was wrong claiming those are unused - they are copied by build.* scripts.
They might be needed when we do the full build for defaults (unless we unvendor vcpkg completely). Currently used for windows as we still use vcpkg for libmagic.

I've updated the PR comment with this info.

Reply:
awesome! Thank you!